zerver/openapi/zulip.yaml

```diff
@@ -19490,6 +19524,10 @@ components:
             the context of a given Zulip server.
 
             E.g. on `example.zulip.com`, this will be `example`.
+        deactivated:
+          type: boolean
+          description: |
+            Only present for stream messages; indicating whether the stream is deactivated.
```

Comment:
So these API documentation changes should go in the same commit where you added it to the actual API.

Additionally, they should have `**Changes**` entries with an API feature level bump; `git log api_docs/` should find you lots of handy examples. See also https://zulip.readthedocs.io/en/latest/documentation/api.html and the new application feature tutorial for examples.

Reply:
Done.